.github/workflows/prepare-release.yml

```
@@ -0,0 +1,79 @@
+name: Prepare release
+
+on:
+  workflow_dispatch:
+
+jobs:
+  prepare-release:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_TOKEN: ${{ secrets.TREZOR_BOT_TOKEN }}
+      FILEPATH: ./packages/suite/package.json
+      MAIN_BRANCH: develop
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: ${{ env.MAIN_BRANCH }}
+
+      - name: Calculate versions
+        id: calculate-versions
+        run: |
+          currentYear=$(date +%y)
+          currentMonth=$(date +%-m)
```

```suggestion
          currentMonth=$(date +%m)
```
I would slightly prefer branch name like release/23.04 over release/23.4 because of the ordering, but doesn't matter that much

---

I did it this way to match the versioning and save some code, but not a problem to change it back.

---

let's leave it, it's simpler 👌